create_framework/dependency_injection.rst

```diff
@@ -99,32 +99,32 @@ Create a new file to host the dependency injection container configuration::
     use Symfony\Component\EventDispatcher;
     use Simplex\Framework;
 
-    $sc = new DependencyInjection\ContainerBuilder();
-    $sc->register('context', Routing\RequestContext::class);
-    $sc->register('matcher', Routing\Matcher\UrlMatcher::class)
+    $container = new DependencyInjection\ContainerBuilder();
```

Comment:
`$containerBuilder`. I would review the usage of `$container` elsewhere as `ContainerBuilder` is very different from `$container`

Reply:
I agree but then we come back here https://github.com/symfony/dependency-injection/blob/master/Extension/Extension.php#L80 where the type is `ContainerBuilder` and name is `container`

Reply:
IMO that should be changed and it's very confusing :/

Reply:
I'm a little confused. I thought ContainerBuilder (besides the misleading name) would be a Container too, as it extends from Container.